README.md

```diff
@@ -39,41 +41,67 @@ we would like things to flow.
 ### Requirements
 
 * [Apache Maven 3.3.3+](https://maven.apache.org/install.html)
-* [Java 8+](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+* OpenJDK 8
 * Network access to https://repository.sonatype.org/content/groups/sonatype-public-grid
 
 Also, there is a good amount of information available at [Bundle Development](https://help.sonatype.com/display/NXRM3/Bundle+Development#BundleDevelopment-BundleDevelopmentOverview)
 
-NOTE: Version 0.0.2+ of Conan is only compatible with Nexus Repo 3.11.0 or greater due to adding Tree View Filtering. If 
-you need to use this plugin with an older version, please use 0.0.1
-
 ### Building
 
 To build the project and generate the bundle use Maven
 
-    mvn clean package -PbuildKar
+    mvn clean install
 
 If everything checks out, the bundle for Conan should be available in the `target` folder
 
-#### Build with Docker
+## Using Conan With Nexus Repository Manager 3
```

Comment:
Trivial: Unless we're capping all title words, I think "With" should be "with".  If we are, then the next header is lower case ("with").

Reply:
fixed 9256cd94a7e43a6f43ad6f942b7665f0468ccb7d